packages/jazz-tools/src/coValues/coStream.ts

```diff
@@ -227,15 +300,23 @@ export class CoStream<Item = any> extends CoValueBase implements CoValue {
     return subscribeToCoValue<S, Depth>(this, id, as, depth, listener);
   }
 
-  /** @category Subscription & Loading */
+  /**
+   * Ensure a `CoStream` is loaded
```

Comment:
Make clearer here that this returns a new instance of the same CoFeed that's deeply loaded as per the spec, or undefined if it can't be loaded that deeply

Reply:
Addressed now